src/nvim/testdir/test_cmdline.vim

```diff
@@ -51,6 +51,19 @@ func Test_complete_wildmenu()
   call feedkeys(":e Xdir1/\<Tab>\<Down>\<Up>\<Right>\<CR>", 'tx')
   call assert_equal('testfile1', getline(1))
 
+  +  " <C-J>/<C-K> mappings to go up/down directories when 'wildcharm' is
+  " different than 'wildchar'.
+  set wildcharm=<C-Z>
```

Comment:
`wildcharm` option is set to non-zero so https://github.com/neovim/neovim/commit/bed9839f46047abd81f7c9b56936ba177829221c won't work here.

Reply:
bed9839 should always work as it is enabled unconditionally.

Reply:
`wildchar` is also non-zero by default.

Reply:
still, bed9839 keeps working.

Reply:
The test fails without the C code changes.